GitIQT/Scenarios/ChangeDirectory.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace GitIQT.Scenarios
+{
+    internal class ChangeDirectory : IScenario
+    {
+        private string Response = string.Empty;
+
+        private string Answer = string.Empty;
+
+        // Ask user to commit changes to their branch
+        public void AskPrompt()
+        {
+            // Ask user to commit changes to their branch
+            Answer = $"cd ProjectBacon";
+
+            var prompt = "However, while the repository has been cloned, you are not currently in it. What do you need to do next to access the local repository?";
+
```

Comment:
Tell the user what directory they are currently in and where they should be after a clone.

![image](https://user-images.githubusercontent.com/26447240/214760651-39326767-b7cc-4e32-ab7f-df5151612b13.png)

Reply:
Do they know that they should be in Project Bacon?